README.md

```diff
@@ -1,34 +1,22 @@
 # OPDB
+This is the database patch repository for the Open Perpetuum Server.  
 
-## DB VERSION CONTROL PROCEDURE
+This repository and others in this github group is developed under the direction of the game design and development team the [Open Perpetuum Project](https://openperpetuum.com): 100% volunteer opensource player development and persistant Perpetuum Server hosting project.
```

Comment:
This repository is developed under the direction of the Open Perpetuum Project game design and development team; a 100% volunteer opensource player development and persistant Perpetuum Server hosting project.